src/stack_mitosis/lookup.clj

```diff
@@ -33,6 +33,64 @@
       (and (or (seq? v) (vector? v))
            (empty? v))))
 
+(defn same-vpc? [db-a db-b]
+  (= (get-in db-a [:DBSubnetGroup :VpcId]) (get-in db-b [:DBSubnetGroup :VpcId])))
+
+(defn restore-snapshot-attributes
+  "Creates a list of additional attributes to clone from original instance into
+  the newly created replica instance.
+
+  https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBInstanceFromDBSnapshot.html
+  has more information on these attributes."
+  [original tags]
+  (let [attributes-to-clone
+        [:CopyTagsToSnapshot
+         :PubliclyAccessible
+         :AutoMinorVersionUpgrade
+         :DBInstanceClass
+         ;; :DeletionProtection ; must be false for repeated invocation
+         ;; :KmsKeyId ;; not supported by restore or modify
+         ;;              but is guaranteed to remain the same
+         ;;              it can only change when we copy a snapshot
+         ;; :SourceRegion ; not applicable?
+         :ProcessorFeatures
+         ;; :UseDefaultProcessorFeatures ; just copy features directly?
+         :Iops
+         :StorageType
+         :MultiAZ]
```

Comment:
Definitely not worth holding this up, but I am a little nervous about this copy pasting. My guess is there is a more restricted list of keys for snapshot, but this is unfortunately one of the places where it's only testable by running it against Amazon. My assumption is that some of these keys are returning nil and so it's working but the code gives the impression it would copy the value when it's not clear if it actually would.  I'll see what I can do about consolidating this, but might require some careful testing in the future.

Reply:
Oh, I actually checked them against the [RestoreDBInstanceFromDBSnapshot](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBInstanceFromDBSnapshot.html) API inputs. It's why I created a separate one. I would have reused the `clone-replica-attributes` otherwise.

Reply:
Oh ok, I guess enough looked copy pasted, I wasn't sure. Happy if it was directly tested!